### ISSUE TITLE
Differentiate pinned accounts from "program change" accounts more

### ISSUE THREAD

**Issue by nathanleclaire** [workbenchapp/solana-workbench#174]
I don't like that the pinned accounts mush together with the program changes accounts that are coming in... what can we do to differentiate them more?

![image](https://user-images.githubusercontent.com/1476820/177651628-6ff7f410-2e61-4da5-83bc-c5ef273fcafd.png)

**Comment by SvenDowideit** [comment]
yeah, I often have 20 pinned accounts, and use them on all nets - but if you're pinning things you only care about for specific debug sessions, then you need a way to group, and fold them up (or hide them)
